spec/jobs/browse_everything_ingest_job_spec.rb

```diff
@@ -11,4 +12,133 @@
       expect(Valkyrie.logger).to have_received(:warn).with("Unable to find resource with ID: bla")
     end
   end
+
+  context "when called with a container resource" do
+    let(:scanned_resource) { FactoryBot.create_for_repository(:scanned_resource) }
+    let(:pending_upload_attributes) do
+      {
+        id: SecureRandom.uuid,
+        created_at: Time.current.utc.iso8601,
+        file_name: "A shared test folder",
+        local_id: "1XzEWzaluDQj0qIH4xqcgxQXcr6",
+        url: "https://www.googleapis.com/drive/v3/files/1XzEWzaluDQj0qIH4xqcgxQXcr6?alt=media",
+        file_size: 0,
+        auth_token: "ya29.GltkBysb2tnNUmUscg0xL1BrXe3g-vALh4Xo5fXrPfOSpJLaPNxfxThztNi8Qdl3uBkEomFis8HeLDpsYb3tKMH1Kf6Zsih63J8HN0KowPHUbVM__",
+        auth_header: JSON.generate("Authorization" => "Bearer ya29.GltkBysb2tnNUmUscg0xL1BrXe3g-vALh4Xo5fXrPfOSpJLaPNxfxThztNi8Qdl3uBkEomFis8HeLDpsYb3tKMH1Kf6Zsih63J8HN0KowPHUbVM__"),
+        type: "application/x-directory",
+        provider: "google_drive"
+      }
+    end
+    let(:pending_upload) do
+      PendingUpload.new(pending_upload_attributes)
+    end
+    let(:pending_uploads) { [pending_upload] }
+    let(:pending_upload_ids) { pending_uploads.map(&:id).map(&:to_s) }
+    let(:adapter) { Valkyrie::MetadataAdapter.find(:indexing_persister) }
+    let(:storage_adapter) { Valkyrie::StorageAdapter.find(:disk_via_copy) }
+    let(:change_set_persister) do
+      ChangeSetPersister.new(metadata_adapter: adapter, storage_adapter: storage_adapter)
+    end
+    let(:provider) { instance_double(BrowseEverything::Driver::GoogleDrive) }
+    let(:provider_class) { class_double(BrowseEverything::Driver::GoogleDrive) }
+    let(:retriever) { instance_double(BrowseEverything::Retriever) }
+    let(:auth_token) do
+      "ya29.GltkBysb2tnNUmUscg0xL1BrXe3g-vALh4Xo5fXrPfOSpJLaPNxfxThztNi8Qdl3uBkEomFis8HeLDpsYb3tKMH1Kf6Zsih63J8HN0KowPHUbVM__"
```

Comment:
is this a real token? Maybe name it something that indicates it's fake.

Reply:
These were updated to use the string `fake_token`, please see : https://github.com/pulibrary/figgy/pull/3290/files#diff-6e1d982e3b667f79274c6a15bda715b3R26 ,https://github.com/pulibrary/figgy/pull/3290/files#diff-6e1d982e3b667f79274c6a15bda715b3R46 , and https://github.com/pulibrary/figgy/pull/3290/files#diff-6e1d982e3b667f79274c6a15bda715b3R105